content/docs/tasks/security/vault-ca/index.md

```diff
@@ -69,53 +66,74 @@ env:
+
+1.  Edit the service account `vault-citadel-sa` to use an example JWT token that has been configured
+on the testing Vault CA. The reason of this configuration is that a Vault CA requires authentication
+and authorization of Kubernetes service accounts.
+For the information about configuring Vault for Kubernetes authentication and authorization,
+please refer to [Vault Kubernetes auth method](https://www.vaultproject.io/docs/auth/kubernetes.html).
+
+    {{< text bash >}}
+    $ export SA_SECRET_NAME=$(kubectl get serviceaccount vault-citadel-sa -o=jsonpath='{.secrets[0].name}')
+    $ kubectl edit secret ${SA_SECRET_NAME}
+
+    # When editing the secret, change the field "token" to: ZXlKaGJHY2lPaUpTVXpJMU5pSXNJbXRwWkNJNklpSjkuZXlKcGMzTWlPaUpyZFdKbGNtNWxkR1Z6TDNObGNuWnBZMlZoWTJOdmRXNTBJaXdpYTNWaVpYSnVaWFJsY3k1cGJ5OXpaWEoyYVdObFlXTmpiM1Z1ZEM5dVlXMWxjM0JoWTJVaU9pSmtaV1poZFd4MElpd2lhM1ZpWlhKdVpYUmxjeTVwYnk5elpYSjJhV05sWVdOamIzVnVkQzl6WldOeVpYUXVibUZ0WlNJNkluWmhkV3gwTFdOcGRHRmtaV3d0YzJFdGRHOXJaVzR0Y21aeFpHb2lMQ0pyZFdKbGNtNWxkR1Z6TG1sdkwzTmxjblpwWTJWaFkyTnZkVzUwTDNObGNuWnBZMlV0WVdOamIzVnVkQzV1WVcxbElqb2lkbUYxYkhRdFkybDBZV1JsYkMxellTSXNJbXQxWW1WeWJtVjBaWE11YVc4dmMyVnlkbWxqWldGalkyOTFiblF2YzJWeWRtbGpaUzFoWTJOdmRXNTBMblZwWkNJNklqSXpPVGs1WXpZMUxUQTRaak10TVRGbE9TMWhZekF6TFRReU1ERXdZVGhoTURBM09TSXNJbk4xWWlJNkluTjVjM1JsYlRwelpYSjJhV05sWVdOamIzVnVkRHBrWldaaGRXeDBPblpoZFd4MExXTnBkR0ZrWld3dGMyRWlmUS5STkgxUWJhcEpLUG1rdFYzdENucGl6N2hvWXB2MVRNNkxYelRoT3RhRHA3TEZwZUFOWmNKMXpWUWR5czNFZG5sa3J5a0dNZXBFanNkTnVUNm5kSGZoOGpSSkFadU5XTlBHcmh4ejRCZVVhT3FaZzN2N0F6SmxNZUZLallfZmlUWVlkMmdCWlp4a3B2MUZ2QVBpaEhZbmcyTmVOMm5LYmlaYnNuWk5VMXFGZHZiZ0NJU2FGcVRmMGRoNzVPemdDWF8xRmg2SE9BN0FOZjdwNTIyUERXX0JSbG4wUlR3VUpvdkNwR2VpTkNHZHVqR2lOTERaeUJjZHRpa1k1cnlfS1hUZHJWQWNUVXZJNmx4d1JiT05OZnVOOGhySURsOTV2SmpoVWxFLU8tX2N4OHFXdFhOZHFKbE1qZTFTc2lQQ0w0dXE3ME9lcEdfSTRhU3pDMm84YUR0bFE=
     {{< /text >}}
 
-1.  List Node Agent's pods:
+1.  Deploy the example backends:
 
     {{< text bash >}}
-    $ kubectl get pod -n istio-system -l app=nodeagent -o jsonpath={.items..metadata.name}
+    $ kubectl apply -f httpbin-injected.yaml
+    $ kubectl apply -f sleep-injected.yaml
     {{< /text >}}
 
-1.  View each Node Agent's logs. The Node Agent residing on
-the same node as the testing workload will contain Vault related logs.
+## Istio mutual TLS with Vault CA integration
+
+This section provides a demo of Istio mutual TLS with Vault CA integration. With the previous steps,
+mutual TLS is enabled for the Istio deployment and the testing workloads `httpbin` and `sleep` receive
+certificates from the testing Vault CA. When sending a curl request from the `sleep` workload
+to the `httpbin` workload, the request goes through a mutual TLS protected channel constructed from
+the certificates issued by the Vault CA.
+
+1.  Send a `curl` request from the `sleep` workload to the `httpbin` workload.
+The request should succeed with a 200 response code since it goes through a mutual TLS protected channel
```

Comment:
Remember to indent properly when you have steps with multiple lines. Please move this text after the command. That way, you provide readers with a clear logical chain.

```suggestion
    The request succeeds with a `200` response code if the Vault CA constructed the
    mutual TLS protected channel properly.
```

Reply:
Done.